storage/src/common/list_result.cc

```diff
@@ -0,0 +1,212 @@
+// Copyright 2021 Google LLC
```

Comment:
Wrong year, please fix this in all files you've added, even ones I don't comment on.